sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -110,6 +118,10 @@ class TextSuite extends QueryTest with SharedSQLContext {
     Thread.currentThread().getContextClassLoader.getResource("text-suite.txt").toString
   }
 
+  private def testDir: String = {
```

Comment:
actually this is a little bit far from the test case. since it is only used once, why not move this into the test case?